include/cscore.h

```diff
@@ -1,12 +1,12 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) FIRST 2016. All Rights Reserved.                             */
+/* Copyright (c) 2016-2017 FIRST. All Rights Reserved.                        */
 /* Open Source Software - may be modified and shared by FRC teams. The code   */
 /* must be accompanied by the FIRST BSD license file in the root directory of */
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-#ifndef CSCORE_H_
-#define CSCORE_H_
+#ifndef INCLUDE_CSCORE_H_
+#define INCLUDE_CSCORE_H_
```

Comment:
That doesn't make sense, particularly in our case.  With the new repo layout (see wpiutil), this would mean header guards like `SRC_MAIN_NATIVE_INCLUDE_SUPPORT_BASENAME_H_`.  Please no.